wallets/index.js

```diff
@@ -32,6 +33,22 @@ export function useWallet (name) {
   const hasConfig = wallet?.fields.length > 0
   const _isConfigured = isConfigured({ ...wallet, config })
 
+  const enablePayments = useCallback(() => {
+    enableWallet(name, me)
+    logger.ok('payments enabled')
+  }, [name, me, logger])
+
+  const disablePayments = useCallback(() => {
+    disableWallet(name, me)
+    logger.info('payments disabled')
+  }, [name, me, logger])
+
+  if (wallet) {
+    wallet.isConfigured = _isConfigured
+    wallet.enablePayments = enablePayments
+    wallet.disablePayments = disablePayments
+  }
```

Comment:
Mhh, maybe not the best code style but this makes `wallet.enablePayments()` etc. available in `useConfig` even though `wallet` is passed before this code runs :thinking: